src/components/App-footer/app-footer.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
Please, lowercase the name of the folder

Reply:
Resolved